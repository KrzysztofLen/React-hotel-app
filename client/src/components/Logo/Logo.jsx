// @flow
import React from 'react';
import {NavLink} from 'react-router-dom'
import logo from '../../assets/SVG/logo.png';

export const Logo = () => (
	<div className={"logo__container"}>
		<NavLink to="/">
			<img src={logo} className="logo" alt="trillo logo"/>
		</NavLink>
	</div>
);