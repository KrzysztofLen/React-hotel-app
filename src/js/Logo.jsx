import React from 'react';
import { NavLink } from 'react-router-dom'
import logo from '../assets/logo.png';

export const Logo = () => (
	<NavLink to="/">
		<img src={logo} className="logo" alt="trillo logo"/>
	</NavLink>
);