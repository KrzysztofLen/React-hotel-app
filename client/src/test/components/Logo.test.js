import React from 'react';
import { shallow } from 'enzyme';
import { Logo } from '../../components/Logo/Logo';

test('should render Logo correctly', () => {
  const wrapper = shallow(<Logo />);
  expect(wrapper.debug()).toMatchSnapshot();
});
