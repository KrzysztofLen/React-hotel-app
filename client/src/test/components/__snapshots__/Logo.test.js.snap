// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Logo correctly 1`] = `
<NavLink
  activeClassName="active"
  aria-current="page"
  to="/"
>
  <img
    alt="trillo logo"
    className="logo"
    src={null}
  />
</NavLink>
`;
